.github/workflows/tests_without_binary.yml

```diff
@@ -87,7 +87,7 @@ jobs:
         if: inputs.pennylane-version == 'latest'
         run: |
           cd pennylane
-          git checkout master
+          git checkout dq-defer-measurements-shots
```

Comment:
I think the correct place to update the branch is in `requirements-dev.txt`. This file is for workflows for testing devices when their binaries are not installed, i.e, nothing is supposed to work.

Reply:
Thanks.